spec/unit/config_spec.rb

```diff
@@ -72,6 +72,21 @@ class Hiera
           Config.load('/dev/null')
         end
       end
+
+      describe "if deep_merge can't be loaded" do
+        let(:error_message) { "Must have 'deep_merge' gem installed for the configured merge_behavior." }
+        before(:each) do
+          Config.expects(:require).with("deep_merge").raises(LoadError, "unable to load")
```

Comment:
This causes `require "deep_merge"` to raise a LoadError, which is what's rescued above. My expectation below isn't specific about what type of error it expects, so I'll add that.